flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/AccessExecutionGraph.java

```diff
@@ -176,6 +176,13 @@ public interface AccessExecutionGraph extends JobStatusProvider {
      */
     Optional<String> getCheckpointStorageName();
 
+    /**
+     * Returns whether the state changelog is enabled for this ExecutionGraph.
+     *
+     * @return true, if state changelog enabled, false otherwise.
+     */
+    boolean isStateChangelogEnabled();
```

Comment:
I think `isChangelogStateBackendEnabled` looks better.

Reply:
Hi @Myasuka , thanks for the suggession .I think ’State Changelog‘ is more suitable to describe this feature, and ’ChangelogStateBackend‘ is the specific implementation of it. WDYT ?

Reply:
I think isStateChangelogEnabled is easier to understand, but I've also noticed that isChangelogStateBackendEnabled has been used elsewhere. I don't know how to choose, or is my understanding simply wrong?

Reply:
To me, making it consistent with the other parts of code makes sense (so `isChangelogStateBackendEnabled` seems preferrable); otherwise, I don't have a strong opinion on this.

Reply:
Actually I think all place where `isChangelogStateBackendEnabled` appeared should be `isStateChangelogEnabled`. But since these methods are not public API for users , I can also accept `isChangelogStateBackendEnabled` for consistence.